src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -32,7 +34,7 @@ namespace traits
 {
     /** Trait for int */
     template<>
-    struct PICToAdios<int>
```

Comment:
Well it has a) to be merged and b) pass the tests on the CI suite. So I don't think it would compile. (How could it, if it conflicts?) And if it even crashes then it is a bug introduced by the developer and undetected by the reviewer.
I just think that a comment like this is nowhere else and if we start adding comments with coding guidelines to the code there will be quite some code bloat with something that should be obvious or (if at all) documented in the general coding guidelines.